chain/blockValidator.go

```diff
@@ -308,11 +307,11 @@ func SignerCheck(header *block.Header) error {
 
 	rawPubKey, err := crypto.DecodePoint(publicKey)
 	if err != nil {
-		return err
+		return fmt.Errorf("decode public key error: %v", err)
 	}
 	err = crypto.Verify(*rawPubKey, signature.GetHashForSigning(header), header.Signature)
 	if err != nil {
-		return err
+		return fmt.Errorf("invalid header signature %x", header.Signature)
```

Comment:
@yilunzhang 
Suggestion:  Don't override err info from crypto.Verify. Just extend err info.

Reply:
You are right, let me correct it in the current PR